### ISSUE TITLE
Unable to download container ssh key

### ISSUE THREAD

**Issue by Atalanta** [Atalanta/cucumber-chef#83]
Downloading container SSH credentials...//var/lib/gems/1.9.1/gems/net-sftp-2.0.5/lib/net/sftp/operations/download.rb:307:in `on_open': open /home/ubuntu/.ssh/id_rsa: no such file (2│
) (RuntimeError)                                                                                                                                                                     │
        from /var/lib/gems/1.9.1/gems/net-sftp-2.0.5/lib/net/sftp/request.rb:87:in`call'                                                                                            │
        from /var/lib/gems/1.9.1/gems/net-sftp-2.0.5/lib/net/sftp/request.rb:87:in `respond_to'                                                                                      │
        from /var/lib/gems/1.9.1/gems/net-sftp-2.0.5/lib/net/sftp/session.rb:948:in`dispatch_request'                                                                               │
        from /var/lib/gems/1.9.1/gems/net-sftp-2.0.5/lib/net/sftp/session.rb:911:in `when_channel_polled'                                                                            │
        from /var/lib/gems/1.9.1/gems/net-ssh-2.2.2/lib/net/ssh/connection/channel.rb:311:in`call'                                                                                  │
        from /var/lib/gems/1.9.1/gems/net-ssh-2.2.2/lib/net/ssh/connection/channel.rb:311:in `process'                                                                               │
        from /var/lib/gems/1.9.1/gems/net-ssh-2.2.2/lib/net/ssh/connection/session.rb:214:in`block in preprocess'                                                                   │
        from /var/lib/gems/1.9.1/gems/net-ssh-2.2.2/lib/net/ssh/connection/session.rb:214:in `each'                                                                                  │
        from /var/lib/gems/1.9.1/gems/net-ssh-2.2.2/lib/net/ssh/connection/session.rb:214:in`preprocess'                                                                            │
        from /var/lib/gems/1.9.1/gems/net-ssh-2.2.2/lib/net/ssh/connection/session.rb:197:in `process'                                                                               │
        from /var/lib/gems/1.9.1/gems/net-ssh-2.2.2/lib/net/ssh/connection/session.rb:161:in`block in loop'                                                                         │
        from /var/lib/gems/1.9.1/gems/net-ssh-2.2.2/lib/net/ssh/connection/session.rb:161:in `loop'                                                                                  │
        from /var/lib/gems/1.9.1/gems/net-ssh-2.2.2/lib/net/ssh/connection/session.rb:161:in`loop'                                                                                  │
        from /var/lib/gems/1.9.1/gems/net-sftp-2.0.5/lib/net/sftp/session.rb:802:in `loop'                                                                                           │
        from /var/lib/gems/1.9.1/gems/net-sftp-2.0.5/lib/net/sftp/operations/download.rb:186:in`wait'                                                                               │
        from /var/lib/gems/1.9.1/gems/net-sftp-2.0.5/lib/net/sftp/session.rb:125:in `download!'                                                                                      │
        from /var/lib/gems/1.9.1/gems/cucumber-chef-2.0.3/lib/cucumber/chef/ssh.rb:125:in`download'                                                                                 │
        from /var/lib/gems/1.9.1/gems/cucumber-chef-2.0.3/lib/cucumber/chef/provisioner.rb:136:in `block (2 levels) in download_proxy_ssh_credentials'                               │
        from /var/lib/gems/1.9.1/gems/cucumber-chef-2.0.3/lib/cucumber/chef/provisioner.rb:134:in`each'                                                                             │
        from /var/lib/gems/1.9.1/gems/cucumber-chef-2.0.3/lib/cucumber/chef/provisioner.rb:134:in `block in download_proxy_ssh_credentials'                                          │
        from /var/lib/gems/1.9.1/gems/cucumber-chef-2.0.3/lib/cucumber/chef/utility.rb:109:in`spinner'                                                                              │
        from /var/lib/gems/1.9.1/gems/cucumber-chef-2.0.3/lib/cucumber/chef/provisioner.rb:129:in `download_proxy_ssh_credentials'                                                   │
        from /var/lib/gems/1.9.1/gems/cucumber-chef-2.0.3/lib/cucumber/chef/provisioner.rb:69:in`build'                                                                             │
        from /var/lib/gems/1.9.1/gems/cucumber-chef-2.0.3/bin/cucumber-chef:149:in `setup'                                                                                           │
        from /var/lib/gems/1.9.1/gems/thor-0.16.0/lib/thor/task.rb:27:in`run'                                                                                                       │
        from /var/lib/gems/1.9.1/gems/thor-0.16.0/lib/thor/invocation.rb:120:in `invoke_task'                                                                                        │
        from /var/lib/gems/1.9.1/gems/thor-0.16.0/lib/thor.rb:275:in`dispatch'                                                                                                      │
        from /var/lib/gems/1.9.1/gems/thor-0.16.0/lib/thor/base.rb:425:in `start'                                                                                                    │
        from /var/lib/gems/1.9.1/gems/cucumber-chef-2.0.3/bin/cucumber-chef:453:in`<top (required)>'                                                                                │
        from /usr/local/bin/cucumber-chef:19:in `load'                                                                                                                               │
        from /usr/local/bin/cucumber-chef:19:in`<main>'

**Comment by zpatten** [comment]
That means the ssh-keys didn't get generated when the test lab was provisioned with Chef.  My plan this weekend is to get down into all these recent issues and get them resolved.

**Comment by zpatten** [comment]
I can confirm, just reproduced:

```
Downloading container SSH credentials...//home/zpatten/.rvm/gems/ruby-1.9.3-p194@chef-repo/gems/net-sftp-2.0.5/lib/net/sftp/operations/download.rb:307:in `on_open': open /home/ubuntu/.ssh/id_rsa: no such file (2) (RuntimeError)
```

**Comment by zpatten** [comment]
So it appears the full release was still trying to install the pre-release gem and this was failing since it was scoping the version to "2.0.3" and not "2.0.3.pre" of course.  Doing final testing on it now.

**Comment by zpatten** [comment]
This is fixed in commit e99ad7b0ed582c21e91dd4159cf751990416f315
